{
	"name": "@moll/fetch-throw",
	"version": "0.3.2",
	"description": "A Fetch API mixin to throw FetchError when the request fails or response has an error.",
	"keywords": [
		"error",
		"errors",
		"exception",
		"fetch",
		"http"
	],
	"homepage": "https://github.com/moll/js-fetch-throw",
	"bugs": "https://github.com/moll/js-fetch-throw/issues",

	"author": {
		"name": "Andri MÃ¶ll",
		"email": "andri@dot.ee",
		"url": "http://themoll.com"
	},

	"repository": {
		"type": "git",
		"url": "git://github.com/moll/js-fetch-throw.git"
	},

	"licenses": [{
		"type": "LAGPL",
		"url": "https://github.com/moll/js-fetch-throw/blob/master/LICENSE"
	}],

	"main": "index.js",
	"scripts": {"test": "make test"},

	"dependencies": {
		"fetch-error": ">= 1.0.0 < 2"
	},

	"devDependencies": {
		"mocha": ">= 2.5.3 < 3",
		"co-mocha": ">= 1.0.0 < 2",
		"must": ">= 0.13.2 < 0.14",
		"sinon": ">= 1.15.4 < 2",
		"whatwg-fetch": ">= 0.9.0 < 0.10",
		"require-guard": ">= 1.0.0 < 2"
	}
}
